mapping.js

```diff
@@ -1,14 +1,2 @@
-export {
-  completion,
-  date,
-  flexibleContent,
-  integer,
-  keyword,
-  long,
-  nested,
-  object,
-  repeater,
-  searchAsYouType,
-  text,
-  array,
-} from './src/mapping'
+Object.defineProperty(exports, "__esModule", { value: true });
+Object.assign(exports, require('./lib/mapping.js'));
```

Comment:
Hmm, dit verpest code hinting. Dus ik denk dat we dit moeten doen:

```js
const mapping = require('./lib/mapping.js');
Object.defineProperty(exports, "__esModule", { value: true });
exports.array = mapping.array;
exports.completion = mapping.completion;
exports.date = mapping.date;
exports.flexibleContent = mapping.flexibleContent;
exports.integer = mapping.integer;
exports.keyword = mapping.keyword;
exports.long = mapping.long;
exports.nested = mapping.nested;
exports.object = mapping.object;
exports.repeater = mapping.repeater;
exports.searchAsYouType = mapping.searchAsYouType;
exports.text = mapping.text;
```

Reply:
Is inderdaad geen gek idee. Wellicht dat `module.exports = require('./src/mapping')` dan ook werkt.

Die `__esModule` is wat transpilers standaard doen om te zorgen dat `default` exports goed werken. Maar daar hebben we hier geen, oh wacht, de `mapping` renderer wel.

Dus we zullen ze toch allemaal met de `exports` manier moeten doen om backwards compatible te blijven.